lighthouse-core/report/v2/renderer/performance-category-renderer.js

```diff
@@ -0,0 +1,194 @@
+/**
```

Comment:
since this is a new file, should we update https://github.com/ChromeDevTools/devtools-frontend/blob/master/front_end/audits2/module.json#L20 ?